src/context/DataContextProvider.tsx

```diff
@@ -1,21 +1,31 @@
 import { useState } from 'react'
 import DataContext from './data-context'
 import { parDataTypes } from '../interfaces/parDataTypes'
+import authHeader from '../helpers/authHeader'
+import axios from 'axios'
 
 const DataContextProvider = ({ children }: { children: React.ReactNode }) => {
   const [data, setData] = useState<parDataTypes[]>([])
   const fetchParticipants = async () => {
     try {
       //'https://semicolon-registration-backend.onrender.com/participants/getAll',
-      const res = await fetch(
-        'https://semicolon-registration-backend.onrender.com/participants/getAll'
-      )
-
-      const participants = await res.json()
-      if (participants.status === 'failure') {
+      // const res = await fetch(
+      //   'https://semicolon-registration-backend.onrender.com/participants/getAll'
+      // )
+      const hdrs = authHeader()
+      if (hdrs) {
+        const res = await axios.get(
+          'https://semicolon-registration-backend.onrender.com/participants/getAll',
+          { headers: hdrs }
+        )
+        const participants = res.data
+        if (participants.status === 'failure') {
```

Comment:
Make the check based on the correct http response unauthenticated code, not data